lib/flipper/ui/views/feature.erb

```diff
@@ -225,7 +225,7 @@
           <div class="row">
             <% unless @feature.boolean_value %>
              <div class="col">
-                <button type="submit" name="action" value="Enable" class="btn btn-outline-success btn-block">
+                <button type="submit" name="action" value="Enable" id="confirm-warn-enable" class="btn btn-outline-success btn-block">
```

Comment:
It might be better to make this `id` a little more generic. Something like `enable_feature__button`. That would stay consistent with some of the other `id` attributes on buttons.

Reply:
Hi @thetimbanks changed the id as requested, it truly makes sense! Can i please get it merged, i believe it would be of great help for our product, if this small change could merge.

Reply:
Reminder @thetimbanks for merging the PR, cc @rsanheim @bkeepers @delynn